core/src/runtime/system.rs

```diff
@@ -642,6 +643,49 @@ impl KompactSystem {
         c.enqueue_control(ControlEvent::Start);
     }
 
+    /// Subscribes the given component to the systems `NetworkStatusPort`.
+    ///
+    /// The status port pushes notifications about the systems Network layer, when new connections
+    /// are set-up, when connections are lost, closed, or dropped. It may also receive requests
+    /// for information about what remote systems the local system is currently connected to.
+    ///
+    /// The `component` must implement [Require](component::Require) for
```

Comment:
Did you double check that these links resolve correctly? I'm always a bit apprehensive about unqualified references in docs, because they don't get caught during normal compilation. Gotta look at the output of the doc command...